bundles/org.openhab.binding.tibber/README.md

```diff
@@ -0,0 +1,119 @@
+# Tibber Binding
+
+The Tibber Binding connects to the [Tibber API](https://developer.tibber.com), and use queries at frequent polls to retrieve price and consumption information for users with Tibber subscription, and provides additional live measurements via websocket for users having Tibber Pulse. 
+
+Refresh time (poll frequency) is set manually as part of setup, minimum 1 minute.
+
+If using Tibber Pulse, Pulse will be recognized automatically by the binding if associated with HomeID used for setup, and live data stream will be initiated if Pulse verified present. 
```

Comment:
This one is also long and can use some grammar improvements.

Reply:
Corrected.